src/hooks/core/auth/useUserAuthHook.ts

```diff
@@ -21,13 +21,19 @@ const useUserAuthHook = () => {
       anonymousToken: null
     });
 
+  const isMe = (profileId: string | undefined): boolean => {
+    if (!profileId) return false;
+    return profileId === signedProfileId || profileId === anonProfileId;
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isMe` function checks if the provided profileId matches either the signedProfileId or anonProfileId. This is a good addition as it provides a way to check if the current user is the same as the one identified by the provided profileId. However, it would be better to add some error handling for cases where both `signedProfileId` and `anonProfileId` are undefined or null.
```diff
  const isMe = (profileId: string | undefined): boolean => {
    if (!profileId || (!signedProfileId && !anonProfileId)) return false;
    return profileId === signedProfileId || profileId === anonProfileId;
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->